common/include/common/common_utils.hpp

```diff
@@ -423,15 +421,15 @@ static inline std::vector<blas3_param_t<scalar_t>> get_blas3_params(
   if (args.csv_param.empty()) {
     warning_no_csv();
     std::vector<blas3_param_t<scalar_t>> blas3_default;
-    constexpr index_t dmin = 64, dmax = 1024;
+    constexpr index_t dmin = 32, dmax = 8192;
     std::vector<std::string> dtranspose = {"n", "t"};
     scalar_t alpha = 1;
     scalar_t beta = 0;
```

Comment:
All looks good to me, I was just wondering if we want to keep `beta = 0` here or change it to 1 as we did for level 2?

Reply:
Ooh, I missed that one, sorry. It is fixed now. Thanks for bringing it up.